egs/wsj/s5/steps/segmentation/lats_to_targets.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+set -o pipefail
+
+silence_phones=
+garbage_phones=
+max_phone_duration=0.5
+acwt=0.1
+
+cmd=run.pl
+
+[ -f ./path.sh ] && . ./path.sh
+. utils/parse_options.sh
+
+if [ $# -ne 4 ]; then
+  cat <<EOF
+  Usage: steps/segmentation/lats_to_targets.sh <data-dir> <lang> <lattice-dir> <targets-dir>"
+  e.g.: steps/segmentation/lats_to_targets.sh \
+  --silence-phones exp/segmentation1a/silence_phones.txt \
+  --garbage-phones exp/segmentation1a/garbage_phones.txt \
+  --max-phone-duration 0.5 \
+  data/train_split10s data/lang \
+  exp/segmentation1a/tri3b_train_split10s_lats \
+  exp/segmentation1a/tri3b_train_split10s_targets
+
+  note: 
+  garbage_phones.txt can contain ambiguous items like OOV, laugh, spoken noise
+  etc.
+  silence_phones.txt might just contain data/lang/silence_phones.txt 
+  other than the garbage phones.
+EOF
```

Comment:
The note is not very clear. Is this a description of what the files are supposed to contain?